### ISSUE TITLE
v4 beta 7 "domain" must omit scheme

### ISSUE THREAD

**Issue by ajwootto** [auth0/nextjs-auth0#1822]
### Checklist

- [X] The issue can be reproduced in the [nextjs-auth0 sample app](https://github.com/auth0-samples/auth0-nextjs-samples/tree/main/Sample-01) (or N/A).
- [X] I have looked into the [Readme](https://github.com/auth0/nextjs-auth0#readme), [Examples](https://github.com/auth0/nextjs-auth0/blob/main/EXAMPLES.md), and [FAQ](https://github.com/auth0/nextjs-auth0/blob/main/FAQ.md) and have not found a suitable solution or answer.
- [X] I have looked into the [API documentation](https://auth0.github.io/nextjs-auth0/) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0/nextjs-auth0/issues) and have not found a suitable solution or answer.
- [X] I have searched the [Auth0 Community](https://community.auth0.com) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Description

- in V3 the issuer base URL was defined using AUTH0_ISSUER_BASE_URL env var.

In v4 this is changed to AUTH0_BASE_URL (personally I think it should be able to work with the old name too)

However the value also must omit the `https://` because the SDK now adds `https://` to the beginning of the value regardless of whether it's there already. The SDK should be smart enough to only add this if it's required, especially since a failure to configure this properly just ends up displaying this page with no other errors logged:

<img width="681" alt="image" src="https://github.com/user-attachments/assets/d450f6da-ea10-4b24-8d9f-09ea4dc7b4c2">

### Reproduction

1. Configure `AUTH0_BASE_URL` to be `https://<your issuer>`
2. it doesn't work

### nextjs-auth0 version

4.0.0-beta.7

### Next.js version

15

### Node.js version

22

**Comment by guabu** [comment]
Hi @ajwootto 👋 v4 of the SDK does not use a `AUTH0_BASE_URL` environment variable, the list of expected environment variables could be found in the README: https://github.com/auth0/nextjs-auth0/tree/v4?tab=readme-ov-file#2-add-the-environment-variables

For example:

```
AUTH0_DOMAIN=example.us.auth0.com
AUTH0_CLIENT_ID=some-client-id
AUTH0_CLIENT_SECRET=some-client-secret
AUTH0_SECRET=some-secret
APP_BASE_URL=https://example.com
```

**Comment by guabu** [comment]
I agree, we can automatically construct the `AUTH0_DOMAIN` by checking if a scheme was provided or not.

Thanks for the feedback! We'll look into getting that implemented for the next release.

For the time being, the `AUTH0_DOMAIN` should not contain the host (e.g.: `example.us.auth0.com`).

**Comment by nandan-bhat** [comment]
Closing this ticket as this issue has been fixed `4.0.0-beta.9`
